openslides/users/static/templates/users/group-list.html

```diff
@@ -12,51 +15,60 @@
     </div>
     <h1 translate>Groups</h1>
   </div>
+
+  <!--<div os-perms="users.can_manage" class="hint">
+    <translate>Hint: The group 'Registered Users' is special. All other groups (except 'Guests') gain all permissions from 'Registered Users', so the other permissions are additional.</translate>
+    </div>-->
 </div>
 
 <div class="details">
-  <div class="row form-group">
-    <div class="col-sm-4 pull-right">
-      <div class="form-group">
-        <div class="input-group">
-          <div class="input-group-addon"><i class="fa fa-search"></i></div>
-          <input type="text" ng-model="filter.search" class="form-control"
-             placeholder="{{ 'Search' | translate}}">
-        </div>
-      </div>
-    </div>
-  </div>
-
-  <table class="table table-striped table-bordered table-hover">
+  <table class="table groups-table table-bordered">
```

Comment:
Try to use `table-hover` to bring more feedback if user hover over a line - it helps to orient in big/wide tables (with many groups).